src/html/index.html.template

```diff
@@ -32,11 +47,28 @@
         }
       }
       #ha-launch-screen {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        width: 100%;
         height: 100%;
```

Comment:
Quite redundant but not a problem

Reply:
This is just making sure the overlay covers the screen until the UI is loaded below, previously the sidebar and other content rendered around the launch screen, also caused a layout jump when it gets removed